doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2875,8 +2875,15 @@ Sample output using "dataverseAdmin" as the ``identifier``::
       "superuser": true,
       "affiliation": "Dataverse.org"
     }
+    
+Alternatively you may pass a user's api ``token`` to retrieve the same output::
+   
+    GET http://$SERVER/api/admin/authenticatedUsers/token/$token
```

Comment:
Unless I'm missing something, he admin api is blocked off (rightly so), so external tools would not be able to call this. Do we have other user APIs, if so this should go there. (if not, we should create something like "/api/users/"*)

* I've never been a fan of the plural, but we should be consistent with "datasets", "dataverses", etc.

Also, we should be getting the token from the header, like we do from other APIs.

Reply:
My bad, I assumed that since it was getting the same info it should have the same permissions required. I will change/fix.

Reply:
I think as it is, the only token you can get info about is the one you already have, and it really functions like a /api/users/me call. Does anyone need to use one apitoken to check other ones? If not, and if this call should allow a header, then perhaps it shouldn't also have a token path parameter (just the header or &key= query param) to avoid scanners.

Reply:
Yes, I left that part out, but by getting the token from the header, it doesn't need to have a token path parameter.

Reply:
For documentation purposes, we already have a "Get All Notifications by User" endpoint and it probably makes sense to put the new docs about getting your user info near these older docs, or vice versa: https://guides.dataverse.org/en/5.1.1/api/native-api.html#get-all-notifications-by-user

Reply:
As discussed, I moved the endpoint to the api/users path. I also moved the doc as suggested by Phil. that whole native api doc needs some reorganization work especially with respect to user related end points.